stock_picking_purchase_order_link/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Stock Picking Purchase Order Link',
+    'summary': """
+        Link between picking and purchase order""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Eficent Business and IT Consulting Services S.L.,'
+              'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/stock-logistics-workflow',
+    "application": False,
+    "installable": True,
+    'depends': [
+        'purchase'
```

Comment:
You have to add purchase_stock as that module was added in v12.

Your tests are passing because of https://github.com/OCA/stock-logistics-workflow/pull/562/files#diff-dd273efb1c4dfddc20425a7b87ce6478R9

That's why I don't like tests on post_install...

Reply:
@rousseldenis that doesn't work anymore on v12, so that's not the reason why they are passing.

Reply:
What do you mean ???

Tests are ok : https://travis-ci.org/OCA/stock-logistics-workflow/jobs/578365850#L1585

Reply:
> the test logic works with already loaded modules (maybe by luck).

That's the point ! That's why I requested dependency on purchase_stock

Reply:
I mean that they are not executed at the end, as that variables are no longer valid (you can do it, but through tags).

Reply:
> That's the point ! That's why I requested dependency on purchase_stock

Yes, I'm not debating on that. Simply pointing that the reason is not by the `post_install = True` variable (it's useless on v12).

Reply:
> (it's useless on v12).

Ok, was not aware. But it could be the problem.

Reply:
So I undestand that requested chages are remove `post_install = True` in tests file (it's useless), add a dependency on `purchase_stock` and squah all v12 migration commits together.

Reply:
Yes, I think so